harvester/cli.py

```diff
@@ -27,9 +27,11 @@
 @click.option('--format',
               default='oai_dc',
               help='Add metadata type (e.g. mods, mets, oai_dc, qdc)')
+@click.option('--set', prompt='Enter the set',
```

Comment:
I would suggest not using a prompt here. My understanding is that this is indeed optional as this tool is already being used to harvest from ArchiveSpace. Making this a prompt will cause the current workflow to hang on startup because the container will be waiting for input.

Reply:
This is a perfect example of me getting tunnel vision and only thinking about my particular use case, my bad. I'm fixing this now